src/STAT_FrontEnd.h

```diff
@@ -175,17 +164,14 @@ class STAT_FrontEnd
 #endif
 
     public:
-        //! Default constructor
-        /*!
-            Sets trace values to defaults and required variables to unset values
-        */
-        STAT_FrontEnd();
+        //! Factory method.
+        static STAT_FrontEnd* make();
 
         //! Default destructor.
         /*!
             First dumps any performance data then frees up allocated variables.
         */
-        ~STAT_FrontEnd();
```

Comment:
The rest of the API changes here is that some of the very launcher specific functions are now virtual so STAT_lmonFrontEnd and STAT_ctiFrontEnd can provide completely different implementations.  

There are also a few new utility functions where otherwise non-launcher functions rely on discrete bits of launcher specific code.